flink-table/flink-sql-parser/src/main/java/org/apache/flink/sql/parser/ddl/SqlCreateFunction.java

```diff
@@ -51,20 +53,24 @@ public class SqlCreateFunction extends SqlCreate {
 
     private final boolean isSystemFunction;
 
+    private final SqlNodeList jarPaths;
```

Comment:
I suggest to rename it for it may be a file instead of a jar, althogh currently we only support jar.